python/forevervm/forevervm/__init__.py

```diff
@@ -68,7 +68,12 @@ def get_binary():
 
 def run_binary():
     binpath = get_binary()
-    subprocess.run([binpath] + sys.argv[1:])
+
+    env = os.environ.copy()
+
+    env["FOREVERVM_RUNNER"] = "uvx"
+    env["FOREVERVM_SDK"] = "python"
```

Comment:
I think it would be useful to send the SDK as a (hard-coded) header when we make calls directly from the SDK, but I don't understand the point of sending SDK when from the wrapped CLI when we already send the runner? After all, they aren't actually using the python sdk when running a cli command, even if they run it through uvx.

Reply:
oh oops good call